.travis.yml

```diff
@@ -43,28 +43,58 @@ addons:
     - clang
     - cmake
     - cmake-data
-
-
 install:
   - if [[ "$CC" == "gcc" ]]; then export CXX="g++-4.8"; fi
-  - sudo pip install --upgrade -r requirements.txt
+  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then
+    export CONDA_OS=MacOSX;
```

Comment:
I think this would be a bit more readable with some indentation..

Reply:
OK can do - just took some previous versions as a template.